### ISSUE TITLE
WCS VendorSpecificCapabilities Validation failure for GetCapabilities response

### ISSUE THREAD

**Issue by hemant559** [opengeospatial/ets-wcs10#32]
We have declared VendorSpecificCapability element in GetCapabilities response.
Team Engine Test cases for GetCapabilities are getting failed.

VendorSpecificCapabilities Declaration in Capabiliites file is as follows - :
```
<wcs:VendorSpecificCapabilities>
<person xmlns:ogc="http://www.opengis.net/ogc" xmlns:ows="http://www.opengis.net/ows/1.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://122.15.218.129:8081/OGC_WCS_Test/person.xsd"/>
</wcs:VendorSpecificCapabilities>
```

For any Team Engine Test case for GetCapabilities - Test Details show following message :

>    Response from parser wcs1-0-0:XMLValidatingParser.corrigendum1:
>       
>    Messages from parser wcs1-0-0:XMLValidatingParser.corrigendum1:
>         Validation error:
>   cvc-complex-type.2.4.c: The matching wildcard is strict, but no declaration can be found for element 'person'.
> 1 validation error detected.
> 
> 
> Result: Failed

Attaching Capabilities response here.
[WCS_Capabilities_Response.zip](https://github.com/opengeospatial/ets-wcs10/files/1654823/WCS_Capabilities_Response.zip)

The schema at http://schemas.opengis.net/wcs/1.0.0/wcsCapabilities.xsd has listed 
vendorSpecificCapabilities contents to be "<xs:any>" so it should be allowed as valid.

1. please provide a sample to declare vendorSpecificCapability contents in any of the reference implementation
2. or update the schema used by validator to have "processContents" as "skip" or "lax", whichever will help validate vendorSpecificCapability element.

**Comment by hemant559** [comment]
waiting for response...

**Comment by hemant559** [comment]
In addition to above:
we have also declared noNamespaceSchemaLocation for the element person as mentioned in above example. Schema for the "Person" should be used at the time team engine validation.

**Comment by lgoltz** [comment]
Thank you for reporting. We will do further investigation.

**Comment by lgoltz** [comment]
Declaration of VendorSpecificCapabilities looks fine (but the "person" is invalid). 

Fixed with commit https://github.com/opengeospatial/ets-wcs10/commit/eadb86cc417f4e095416f1adbfbc20ee113c0563 by adding `processContents="lax"` in the schemas used for validation. If a schema is available the content will be validated now, otherwise it is considered to be valid.

Another solution would be to enhance the parser/validator to collect all schema files from the document before validation.

Fix was tested locally with http://122.15.218.129:8081/OGC_WCS_Test/OGCReqResDispatcher?request=GetCapabilities&service=WCS. The result  contains 10 more successful tests.

**Comment by dstenger** [comment]
Pull request was reviewed and solves the problem.
